styles/widgets/generic/color-schemes/light/generic.light.less

```diff
@@ -1844,6 +1844,7 @@
 @scheduler-recurence-editor-border-color: darken(@base-element-bg, 5%);
 @SCHEDULER_OTHER_MONTH_CELL_OPACITY: .5;
 @SCHEDULER_FILL_FOCUSED_APPOINTMENT: false;
+@SCHEDULER_TIME_INDICATOR_BACKGROUND_COLOR: lighten(@SCHEDULER_BASE_BORDER_COLOR, 9%);
```

Comment:
This constant should be editable in ThemeBuilder. Also, the name should be changed.